src/routes/ProfilePage/ProfilePage.js

```diff
@@ -96,11 +108,53 @@ export default class ProfilePage extends Component {
   }
 
   renderPreferences() {
-    // TODO: add after userPreferences backend is complete  
+    const prefs = this.state.userPreferences;
+    let animalTypes = prefs.TypeOfAnimal ? JSON.parse(prefs.TypeOfAnimal) : null;
+    let breeds = prefs.Breed ? JSON.parse(prefs.Breed) : null;
+    let more = prefs.More ? JSON.parse(prefs.More) : null;
+
+    // Format age range
+    let ageRange;
+    if (prefs.MinAge && prefs.MaxAge) {
+      ageRange = prefs.MinAge + '-' + prefs.MaxAge;
+    } else if (prefs.MinAge) {
+      ageRange = prefs.MinAge + '+';
+    } else if (prefs.MaxAge) {
+      ageRange = '< ' + prefs.MaxAge;
+    }
+
+    // Format objects
+    if (animalTypes) {
+      animalTypes = animalTypes.map((elem) => {
+        return elem.value;
+      }).join(', ');
+    };
+    if (breeds) {
+      breeds = breeds.map((elem) => {
+        return elem.value;
+      }).join(', ');
+    };
+    if (more) {
+      more = more.map((elem) => {
+        // This will add a space before any capital letter, and then lowercase
+        // Simply for formatting purposes
+        return elem.value.replace(/([a-z])([A-Z])/g, '$1 $2').toLowerCase();
+      }).join(', ');
+    };
+
     return (
       <Section>
         <>
-          <h2>Saved Preferences</h2>
+          <h2>Saved Preferences <span className='btn p-2 text-primary' onClick={this.deleteAllPreferences}><FontAwesomeIcon icon='trash-alt' /></span></h2>
```

Comment:
Maybe add more space <br> between my profile section and saved preferences section?